src/components/components/ebay-tooltip-overlay/index.js

```diff
@@ -0,0 +1,21 @@
+const emitAndFire = require('../../../common/emit-and-fire');
+const template = require('./template.marko');
+
+function getInitialState(input) {
```

Comment:
We can just omit `getInitialState` and `getTemplateData` if they just return the first argument.